.github/workflows/release.yaml

```diff
@@ -1,25 +0,0 @@
-name: Create and publish a Python package to GitHub
```

Comment:
Intentionally removed? I don't see any replacement to publish a release

Reply:
Intentionally removed, yes. I need to sit and spend some time thinking about how to handle automated Poetry releases. This one hasn't actually been working, so no need to keep it around

Reply:
https://github.com/mikejgray/neon-homeassistant-skill/issues/32